Rmarkdown/Single_Ascending_Dose_PK.Rmd

```diff
@@ -27,7 +27,7 @@ xgx_theme_set()
 
 #directories for saving individual graphs
 dirs = list(
-  parent_dir= tempdir(),
+  parent_dir = "Parent_Directory",
```

Comment:
OK, but figures in www folder still have old directory (need to clear cache to update)